### PR TITLE
Fix eslint errors for linebreaks on windows

### PR DESCRIPTION
**Pull request by datoml**
I am not 100% sure if it's really windows but I get this error.
Expected linebreaks to be 'LF' but found 'CRLF'.
So I think this should be ok :).

### CODE REVIEW

**Comment by ctrlplusb** [comment]
Hmmm, is this safe for UNIX style?

**Comment by birkir** [comment]
I think this should rather be discussed at the airbnb eslint project. We are extending it so any changes there will ultimately end up in ours.

Even though I know nothing about the issue, I feel like its not the right place.

**Comment by slevr** [comment]
Just stopping by looking at this starter for the first time and noticed this. As a windows dev, this is quite common for me with just about any starter I've used. But merely changing the airbnb linebreak-style to 'windows' will cause issues for unix devs. There is a pull request on this over at airbnb [https://github.com/airbnb/javascript/pull/1224](url) but not sure it received any or will gain any traction.   One going solution for git repos that have multi-platform devs is to include a .gitattributes file ( e.g. https://github.com/Danimoth/gitattributes/blob/master/Web.gitattributes ) to specify the line-endings for the repo thus preventing git from changing them based on dev platform. This in addition to your current .editorconfig should satisfy most cases for both unix and windows devs.

**Comment by birkir** [comment]
I hear you and have looked at the issue and came to the conclusion that this is just as bad as the previous setting, and even worse.

What you are proposing is to tell eslint that all line-breaks should be **CRLF** but not **LF** and merging this will

1. Cause the same problem for all non-windows users and windows users with `autocrlf=false`, but just the other way around.
2. Conflict will be made when saving files in editors because of https://github.com/ctrlplusb/react-universally/blob/next/.editorconfig#L14

Adding `process.platform === 'win32'` will make this more comfortable, but the issue with editorconfig will still exist.

**Bottom line:** I think we should leave it as-is and rather document how-to for windows users.

**Comment by ctrlplusb** [comment]
@datoml Would you please open an issue about this.  I don't want windows users to feel left behind.  I am anti-fanboyism and can appreciate everyone has their own environment and sometimes it isn't even of their choosing (not to bash windows or anything).  I don't want this to be forgotten though, so would appreciate it if we created an issue.  Perhaps we could generically title it "Windows issues" and then it can be a kind of go-to until we sort them all.

**Comment by datoml** [comment]
Is it possible to create an issue of this pull request? :).
If not I create an issue with a link to this pull request.

**Comment by slevr** [comment]
@birkir > why would you say that this won't cause problems 
maybe you misread or the way I wrote it was confusing, I was stated that this change if implemented  **will** cause issues for unix devs. Regardless, I am agreeing with your statements.

As you said, it's actually a combination of github, editorconfig, eslint rules wanting to see lf and windows natively wanting crlf.  I have forked the repo and added a .gitattributes file to the next branch to try it out - it seems to work just fine on my windows box as it essentially prescribes eol=lf for all text files regardless of what ones global autocrlf is set to. In other words force lf upon checkout and don't let windows convert to crlf for anything.

Of course it is a starter so the assumption is that anyone using it will have to tweak it anyway, so windows users just have to deal with line endings in their own apps their own way including what @datoml mentions.

**Comment by birkir** [comment]
@slevr I am really sorry that I misunderstood, thank you for the clarification.

But I have to agree, I don't want to leave windows users out in the dark.

I am also worried about users (like me) who have configured their system-wide git client to use LF on their windows machine.
